src/popup/router/components/TransferSendForm.vue

```diff
@@ -157,12 +172,39 @@ export default {
         .filter(({ field }) => field === 'address')
         .find((error) => WARNING_RULES.includes(error.rule))?.msg || null;
     },
+    urlStatus() {
+      return this.$store.getters['tipUrl/status'](this.formModel.address);
+    },
   },
   watch: {
     formModel: {
       deep: true,
       handler(val) {
         console.log('formModel watch', val);
+
+        this.isTipUrl = validateTipUrl(val.address);
+        if (!checkAensName(val.address) && this.isTipUrl) {
+          const urlStatus = this.$store.getters['tipUrl/status'](val.address);
```

Comment:
This line is redundant to line 175 (computed property `urlStatus`. So below `this.urlStatus` can be used.